src/AcmLabelsInput/AcmLabelsInput.tsx

```diff
@@ -30,13 +31,14 @@ export function AcmLabelsInput(props: {
     }
 
     function removeLabel(label: string) {
-        props.onChange(props.value.filter((l) => l !== label))
+        /* istanbul ignore next */
```

Comment:
ts-ignore is not allowed per the linting rules; so I can't remove the optional chaining below even though; when this removeLabel is called it's not possible for value to be undefined.